bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -456,6 +472,62 @@ impl TryFrom<&Ed25519AddressDto> for Ed25519Address {
     }
 }
 
+/// Describes an Ed25519 address.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct AliasAddressDto {
+    #[serde(rename = "type")]
+    pub kind: u8,
+    pub id: String,
+}
+
+impl From<&AliasAddress> for AliasAddressDto {
+    fn from(value: &AliasAddress) -> Self {
+        Self {
+            kind: AliasAddress::KIND,
+            id: value.to_string(),
+        }
+    }
+}
+
+impl TryFrom<&AliasAddressDto> for AliasAddress {
+    type Error = Error;
+
+    fn try_from(value: &AliasAddressDto) -> Result<Self, Self::Error> {
+        value
+            .id
+            .parse::<AliasAddress>()
+            .map_err(|_| Error::InvalidSyntaxField("address"))
```

Comment:
Can we add the address type to the errors, for example "alias address" instead of "address" for every address type?

Reply:
lets do that